src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,11 +1,19 @@
 package core.basesyntax;
 
 public class UserService {
+
     public void registerUser(User user) {
 
+        try {
+            if (new PasswordValidator().validate(user.getPassword(), user.getRepeatPassword())) {
```

Comment:
do we need to do this check? What will happen if the result of the check will be `false`? Can we get `false` result?

Reply:
You are right.